chatlearn/runtime/dist_actor.py

```diff
@@ -255,13 +255,27 @@ def call_vllm_engine_remote_funcs(self, func_name, *args, **kwargs):
         results.append(res)
         return results
 
+    def call_vllm_engine_and_workers_remote_funcs(self, func_name, *args, **kwargs):
+        """
+        Call remote functions for vllm_engine + workers.
+        """
+        results = []
+        for actor in self.all_actors:
+            res = self.call_actor_remote_func(actor, func_name, *args, **kwargs)
+            results.append(res)
+        res = self.call_actor_remote_func(self.vllm_engine, func_name, *args, **kwargs)
```

Comment:
‘self.vllm_engine’ is None for case ENABLE_VLLM_MODULE_V2=False. Need to check in func 'call_actor_remote_func'

Reply:
DistVLLMActor is only used for VLLMModuleV2. It's checked in chatlearn/schedule/model_manager.py